src/sentry/api/endpoints/dsym_files.py

```diff
@@ -97,13 +95,6 @@ def post(self, request, project):
         return upload_from_request(request, project=project)
 
 
-class GlobalDSymFilesEndpoint(Endpoint):
-    permission_classes = (SystemPermission,)
```

Comment:
iirc this `SystemPermission` was added purely for this. Can we tear all of that down too?